data/processed/ref/mean_ref.yaml

```diff
@@ -0,0 +1,23 @@
+api: https://results.ref.ac.uk
+api_documentation: https://www.ref.ac.uk/2014/pubs/201401/
+description: These are the results of the Research Excellence Framework, where university departments in various disciplines are assessed on the quality of their research. The latest REF was conducted in 2014 so data is available only for one year
+endpoint_source: https://results.ref.ac.uk/(S(hlvnuqzwkag44jp3df3d4q14))/DownloadFile/AllResults/xlsx
+framework_group: 1
+is_experimental: False
+nuts_level: 2
+source: the data provider: UKRI
```

Comment:
I think this should be `source: https://gtr.ukri.org`

Reply:
This data is not from GtR. I added a link to the website where they distribute these results